emails/views.py

```diff
@@ -391,12 +382,15 @@ def _sns_message(message_json):
         formatted_from_address, to_address, subject,
         message_body, attachments, mail, address
     )
-    print(f'Response: {response}')
     address.num_forwarded += 1
     address.last_used_at = datetime.now(timezone.utc)
     address.save(
         update_fields=['num_forwarded', 'last_used_at']
     )
+    
+    message_removed = remove_message_from_s3()
```

Comment:
This calls `remove_message_from_s3` without any arguments, but the function definition looks like it needs `bucket, object_key`?